dropwizard-jetty/src/main/java/io/dropwizard/jetty/HttpConnectorFactory.java

```diff
@@ -720,7 +720,7 @@ protected String httpConnections() {
         return name(HttpConnectionFactory.class,  bindHost, Integer.toString(port), "connections");
     }
 
-    protected ServerConnector buildConnector(Server server,
+    protected Connector buildConnector(Server server,
```

Comment:
This change modifies the public API and therefore is a breaking change.

Reply:
It isn't actually necessary. I did it because the PR I was using as a baseline was doing it and it led to less code duplication. I can remove this commit if it helps https://github.com/dropwizard/dropwizard/pull/7614/commits/22cc888d7af0792698c32002b4bf897ef80044eb.
Though if java 16 is a blocker for this PR, then might as well keep it.

Reply:
If the change isn't necessary, we shouldn't make it.